app/views/session.py

```diff
@@ -52,20 +52,21 @@ def login():
     except (TypeError, ValueError) as e:
         raise InvalidTokenException from e
 
-    metadata = parse_metadata(decrypted_token)
-    eq_id = metadata['eq_id']
-    form_type = metadata['form_type']
+    eq_id = decrypted_token.get('eq_id')
+    form_type = decrypted_token.get('form_type')
+    language_code = decrypted_token.get('language_code', 'en')
+
+    g.schema = load_schema_from_params(eq_id, form_type, language_code)
+    schema_metadata = g.schema.json['required_metadata']
```

Comment:
Not sure it makes sense to call this `required_metadata`. There isn't any such thing as optional metadata so I'd say it's better to just just call it `metadata`.